.github/workflows/create_pypi_package.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build and publish PyPI artifact
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_and_publish_package:
+    name: Build and publish the PyPi package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v2
+
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: false
+
+      - name: Patch package version
+        run: poetry version patch
+
+      - name: Run Dockerfile to build and publish package into PyPI
+        run: |
+          docker build . -t ignored \
+            --build-arg PYPI_USERNAME=${{ secrets.PYPI_USERNAME }} \
```

Comment:
It'd be great to add those secrets as organization's and remove them as repository's for each project that uses them.
* oddrn-generator
* odd-models-package

Reply:
done